Sources/MapboxCoreNavigation/RouteController.swift

```diff
@@ -622,6 +611,7 @@ extension RouteController: Router {
 
         let routeOptions = routeOptions ?? routeProgress.routeOptions
         routeProgress = RouteProgress(route: route, options: routeOptions)
+        announce(reroute: route, at: location, proactive: false)
```

Comment:
This was the one other place where the did-reroute notification needed to get inlined. It’s weird that an application explicitly setting the route will get a notification back telling it what it just did, but that’s the only way that various parts of MapboxNavigation will be able to respond to the new route being set. As a case in point, the addition of this line fixes the integration test failure.